### PR TITLE
JS Quiz - Friskics-Warren

### PR DESCRIPTION
**Pull request by wmfwarren**
Quiz completed.

### CODE REVIEW

File: quiz.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	let printSpaceString = ""; //init the string
+	let spaceCounter = spaceToPrint - 1; //The formula for white space is total rows - row number + 1
+	while(spaceCounter < numRows) {
+		printSpaceString += ' '; //add whitespace
+		spaceCounter++; //increment
+	}
+	return printSpaceString; //return the string
+}
+//this is the MAIN tree function
+function tree(){
```
Comment (stevebrownlee) [inline quiz.js]: **Requirement not met:** Create a `tree` function that should build a pine tree out of a character in the Chrome dev tools console. _**It accepts a single object as an argument.**_ The object should have two key/value pairs.

File: quiz.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	//console.log("Trigger", event); // test to see if function triggered
+	
+	var mySeed = new Seed(); //create a new seed
```
Comment (stevebrownlee) [inline quiz.js]: Interesting approach, to create a new instance of a Seed function which has some properties on it. Unfortunately, it's using more memory in the browser than is needed for this task.  Try to simplify your solution a bit.

**Comment by stevebrownlee** [comment]
@wmfwarren Added some comments that you'll need to think about before completion of the quiz.  If you have any questions at all, come and discuss with one of us.

**Comment by wmfwarren** [comment]
Hey Steve,

I changed the code to be less memory intensive and added an object to be passed to tree function to fit the specs.

**Comment by stevebrownlee** [comment]
@wmfwarren great job.  Merge it!!  👍

**Pull request merged**